DatadogRUM/Tests/Instrumentation/RUMInstrumentationTests.swift

```diff
@@ -24,8 +24,8 @@ class RUMInstrumentationTests: XCTestCase {
         // Then
         withExtendedLifetime(instrumentation) {
             DDAssertActiveSwizzlings([
-                "UIViewController.viewDidAppear:",
-                "UIViewController.viewDidDisappear:",
```

Comment:
We are no longer keeping a reference to the class, only the method pointer.

Reply:
Does it change anything?

What if we have the same method signature coming from different objects? 🤔

Reply:
No, the signature could be the same for different objects but the method pointer will be different 👍 
The `class` was only use for those assertions but is not necessary for swizzling.